index.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+entries:
+  metrics-server-exporter:
+  - apiVersion: v1
+    appVersion: 0.0.7
+    created: "2021-05-28T14:07:57.63680224+02:00"
+    description: A Helm chart for metrics-server-exporter
+    digest: 5d0871e4155cb739e01858d2300da7708b69cad7f77a21bdd6e5584e23ae1f90
+    name: metrics-server-exporter
+    urls:
+    - https://github.com/SplashThat/metrics-server-exporter/raw/mag/create-repo/pkg/metrics-server-exporter-0.1.0.tgz
```

Comment:
should this be changes to point too master instead?

Reply:
probably, but just after merging at least one version.